json_annotation/lib/src/checked_helpers.dart

```diff
@@ -37,7 +37,7 @@ T $checkedNew<T>(String className, Map map, T constructor(),
 /// `JsonSerializableGenerator.checked` is `true`.
 ///
 /// Should not be used directly.
-T $checkedConvert<T>(Map map, String key, T castFunc(Object value)) {
+T $checkedConvert<T>(Map map, String key, T Function(Object value) castFunc) {
```

Comment:
[nit] Consider dropping the argument name. `value` is a noise word that was necessary with the old syntax, but the new syntax does not require it.

Reply:
ack